stock_move_line_lock_qty_done/demo/demo_company.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="base.main_company" model="res.company">
+        <field name="lock_qty_done">False</field>
+    </record>
```

Comment:
To disable the feature for all companies.
```suggestion
    <function model="res.company" name="write">
        <value model="res.company" search="[]" />
        <value eval="{'lock_qty_done': False}" />
    </function>
```

Reply:
I updated the code.